src/tools/genpolicy/rules.rego

```diff
@@ -1286,8 +1309,10 @@ ExecProcessRequest {
 ExecProcessRequest {
     print("ExecProcessRequest 2: input =", input)
 
-    # TODO: match input container ID with its corresponding container.exec_commands.
-    some container in policy_data.containers
+    # get p_container from state
+    container := get_state_val(input.container_id)
+    count(container) > 0
```

Comment:
Is `count` actually needed here? Judging from a [quick experiment](https://play.openpolicyagent.org/p/SuG3Yf6P8T), I believe that `get_state_val` already fails the check if the id is not found.

Reply:
Removed. Tests pass for invalid `container_id`.